src/formatMarkup.js

```diff
@@ -1,31 +1,60 @@
+// @ts-check
```

Comment:
For testing, since this function now has options that can be passed in, create a new describe block for each setting, and tests in side the describe block for each scenario the setting can be in. Reference #8 for markup examples you can use for the tests.